types/react/test/index.ts

```diff
@@ -48,7 +51,7 @@ const container: Element = document.createElement("div");
 // --------------------------------------------------------------------------
 
 const ClassicComponent: React.ClassicComponentClass<Props> =
-    React.createClass<Props, State>({
+    createReactClass<Props, State>({
```

Comment:
If `createClass` isn't a part of react anymore, no need to test it here?

Reply:
Moved top-level tests to `create-react-class.tests.tsx`.